docs/Maintenance.md

```diff
@@ -44,6 +63,21 @@ Data will only be restored if archives that exist at that location. This means
 you can choose to only restore a certain application by placing its archives in
 an empty directory and running this command on it.
 
+### A possible backup strategy
+
+Assuming most collections don't change after processing them for the first time,
+you can create a full backup once for each collection. Except the so called
+`uploads` collection having `sync = True` in [`liquid.ini`](../liquid.ini): 
+
+```ini
+# Collection corresponding to nextcloud uploads
```

Comment:
i wouldn't copy/paste the config for that here. Instead, i'd change the text to say "Except for the `uploads` collection or any other collection with `sync = True` in liquid.ini, because the data for these is expected to change often."

Reply:
to replace the copy/pasted code section, i'd make a permalink like this: https://github.com/liquidinvestigations/node/blob/40963726bf79d3318496572e41f93543c93132f3/examples/liquid.ini#L252-L256

that way, you highlight the correct lines, while also having the selection work when we change that file.

Reply:
Changed and directly linked into https://github.com/liquidinvestigations/node/blob/40963726bf79d3318496572e41f93543c93132f3/examples/liquid.ini#L252-L256